R/learner_Cubist_regr_cubist.R

```diff
@@ -0,0 +1,84 @@
+#' @title Regression Cubist Learner
+#' @author sumny
+#' @name mlr_learners_regr.cubist
+#'
+#' @template class_learner
+#' @templateVar id regr.cubist
+#' @templateVar caller cubist
+#'
+#' @template seealso_learner
+#' @template example
+#' @export
+LearnerRegrCubist = R6Class("LearnerRegrCubist",
+  inherit = LearnerRegr,
+
+  public = list(
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    initialize = function() {
+      ps = ParamSet$new(
+        params = list(
+          ParamInt$new(id = "committees", lower = 1L, upper = 100L, default = 1L,
```

Comment:
briefly looked at docs, are you sure upper limit is 100?

Reply:
seems like they didn't document this well, but setting committees to > 100 results in `Error: number of committees must be between 1 and 100`